docs/user/guides/technical-docs-seo-guide.rst

```diff
@@ -125,20 +131,47 @@ For example, if you do have a video in your docs,
 make sure the rest of that page describes the content of the video.
 
 When using images, make sure to set the image alt text or set a caption on figures.
-For Sphinx, the image and figure directives support this:
 
-.. sourcecode:: rst
+.. tabs::
+
+   .. tab:: Sphinx
+
+        For Sphinx, the image and figure directives support both alt texts and captions:
 
-    .. image:: your-image.png
-        :alt: A description of this image
+        .. code-block:: rst
 
-    .. figure:: your-image.png
+            .. image:: your-image.png
+                :alt: A description of this image
 
-        A caption for this figure
+            .. figure:: your-image.png
 
+                A caption for this figure
 
-Redirects
-~~~~~~~~~
+   .. tab:: MkDocs
+
+        The Markdown syntax fundamentally prescribes an alt text for images:
```

Comment:
```suggestion
        The Markdown syntax defines an alt text for images:
```

`fundamentally prescribes` sounds like a PhD lecture :) This is a pretty easy one to use simple language for, so let's use it!

Reply:
haha, true, the contrast to the basic syntax providing the alt text is that there isn't anything for figures. Which IMO is a real negative of using MkDocs. I didn't find good solutions for figures, so I can't instruct people "here's an easy way to create and reference a figure".

Reply:
Thanks for writing the suggestion!